packages/slice-machine/lib/models/common/widgets/StructuredText/Form.tsx

```diff
@@ -0,0 +1,222 @@
+import { useEffect, useState } from "react";
+import * as yup from "yup";
+import { createFieldNameFromKey } from "@lib/forms";
+import { CheckBox as CheckBoxConstructor } from "@lib/forms/fields";
+import { DefaultFields } from "@lib/forms/defaults";
+import options, { optionValues } from "./options";
+import WidgetFormField from "@lib/builders/common/EditModal/Field";
+import { Text, Button, Label, Checkbox, Flex, Box } from "theme-ui";
+import { Col, Flex as FlexGrid } from "@components/Flex";
+import SliceMachineIconButton from "@components/SliceMachineIconButton";
+
+import { has } from "fp-ts/Record";
+import { AnyObject } from "yup/lib/types";
+import { isObject, isString } from "lodash";
+
+const isAllSet = (curr: Array<string>): boolean =>
+  !optionValues.find((e) => !curr.includes(e));
+
+const _createInitialOptions = (str: string) => {
+  const arr = str.split(",");
+  return options.reduce<Array<string>>((acc, { value }) => {
+    if (arr.includes(value)) {
+      return [...acc, value];
+    }
+    return acc;
+  }, []);
+};
+
+function getValueFromYupContext(
+  ctx: yup.TestContext<AnyObject>,
+  key: string
+): string | null {
+  if (
+    has("parent", ctx) &&
+    isObject(ctx.parent) &&
+    has(key, ctx.parent) &&
+    isString(ctx.parent[key])
+  )
+    return ctx.parent[key];
+  return null;
+}
+
+function createValidator(key: "single" | "multi") {
+  return () =>
+    yup.string().test({
+      name: key,
+      message: `Options cannot be empty`,
```

Comment:
I tested the PR and it seems to work, I just don't understand how.
This message appears only so it has to be it, but it's not used on the Accept field but on the fields before.

That's why i can't figure out how it's triggered for the Accept field as well.

Reply:
Here: https://github.com/prismicio/slice-machine/pull/605/files#diff-6510a41b30c174d7f6e9682d3fbf894ec27c13b472ce9c8fdc7bf1f606b53e8aR169

And the options in the options.ts file